presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -36,6 +37,20 @@ private static boolean isEnabled(String propertyName, Session session, boolean d
         return Boolean.valueOf(enabled);
     }
 
+    private static int getCount(String propertyName, Session session, int defaultValue)
```

Comment:
maybe name this getValue or getNumber, since it can be used to parse things besides counts